notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -4951,6 +4951,51 @@ lobotomization [Wynn was rendered mentally handicapped my means of a lobotomizat
 duty to disobey illegal orders [Rains called out the low-level Cube operator Wynn for invoking this defense when he justified himself by claiming that he was just the computer guy.]
 
 
+movie: The Final Cut (2004)
+===========================
+
+:: Title
+The Final Cut
+
+:: Description
+The Final Cut is a 2004 science fiction psychological thriller film written
+and directed by Omar Naim. It stars Robin Williams, Jim Caviezel, Mira
+Sorvino, Mimi Kuzyk, Stephanie Romanov, Genevieve Buechner and Brendan
+Fletcher.
+
+Synopsis: A man specializes in editing the memories of unsavory people into
+uncritical memorials that are played at funerals.
+
+:: Date
+2004-10-15
+
+:: References
+https://en.wikipedia.org/wiki/The_Final_Cut_(2004_film)
+
+:: Ratings
+3 <paul>
+
+:: Major Themes
+memory recorder [A special memory recording implant, called the Zoe implant, is a main novelty of the film.]
+remorse [Alan was consumed with guilt over a conviction that as a child he'd left a boy to die in an abandoned factory. Much of the film followed Alan's efforts to uncover the identity of the boy.]
+childhood trauma [Alan was haunted by a childhood memory of having left a boy to die in an abandoned factory. He was motivated throughout the film by a desire to find out the identify of the boy. It was strongly implied that young Isabel was having issues on account that she'd been molested by her father.]
+
+:: Minor Themes
+human childhood [The viewer is shown Alan as a child exploring an abandoned factory together with another boy.]
+domestic violence [An emotionless Alan watched a recording of a dead man's memory of the man beating his own partner.]
+suicide [Hasan told of how a woman jumped out of a 27th floor window to her death.]
+coping with the death of a spouse [Jennifer shed a crocodile tear for her recently deceased husband.]
+master and servant [Jennifer voiced an order to her butler.]
+romantic infidelity [Danny's memory recordings revealed that he'd slept with an acquaintance of his wife.]
+funeral rite [A funeral was held for Danny.]
+social activism [Placard carrying protestors were angry about the prevalence of memory recording devices in society.]
+pedophiles [A viewing of Bannister's recorded memories strongly suggested that he was molesting his young daughter, Isabel.]
+child abuse [A viewing of Bannister's recorded memories strongly suggested that he was molesting his young daughter, Isabel.]
+invasion of privacy [It came to light that Alan had been secretly watching recordings of his love interest, Delila, sleeping with her now deceased ex-lover. The video footage was particularly sensitive as it was made up of her ex-lover's memories. She was furious at Alan for having done so.]
```

Comment:
Is this a case of non consensual voyeurism? Seems like a sexual crime of one sort or another -

Reply:
I was pondering about that myself. Is it [nonconsensual voyeurism](https://www.themeontology.org/theme.php?name=nonconsensual%20voyeurism) when someone secretly watches a prerecorded video footage of another person?

Reply:
Let's go with yes.